src/pages/Home/Home.jsx

```diff
@@ -38,9 +39,18 @@ const entries = [
 ];
 
 const Home = () => {
+
+  const [randint, setRandom] = useState(0);
+  
+  useEffect(() => {
+    setRandom(Math.floor(Math.random() * 100 + 1));
+  }, []);
   return (
     <div className="home">
       <div className="landing">
+
+        <p id="line1">Hello Alex! Welcome to Toody.</p>
+        <p id="line2">You have been using Toody for {randint} days!</p>
```

Comment:
CSS red flag: make sure to use `className` and corresponding CSS selector `.this-is-my-class` instead of id, you can Google why but tldr it's best practices and allows for reusability.

Reply:
Also make sure to have more descriptive names rather than `line1` or `line2`, instead `welcome-text` and `days-used-text`, for example.